app/screens/Onboarding/Parents/SignIn/SelectUser/SelectUser.tsx

```diff
@@ -27,7 +26,9 @@ export const SelectUserScreen: React.FC<ISelectUserScreenProps> = ({
       <BottomButtonView
         buttonTitle={t('buttons.manage_profiles')}
         // TODO: change noop to real function
-        onSubmit={_.noop}>
+        onSubmit={() => {
+          trackButtonPress('manage');
+        }}>
```

Comment:
Please, do not create arrow functions in JSX thread.
Create function before and wrap it with `useCallback`